pkg/types/names.go

```diff
@@ -33,5 +33,5 @@ const (
 	NsxCATempPath               string = "/tmp/nsx.ca"
 	NsxNodeAgentContainerName   string = "nsx-node-agent"
 	OsReleaseFile               string = "/host/etc/os-release"
-	TimeBeforeRecoverNetwork    int64  = 150
+	TimeBeforeRecoverNetwork    int64  = 180
```

Comment:
why increase this time?

Reply:
Because in QE's test, the time between 2 restarts for unhealthy is about 3 min, previous 150 seconds looks not insurable.